Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_12.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### CrowdStrike Falcon - True Positive Incident Handling
+Fixed an issue with the 'ServiceNow' ticketing system.
```

Comment:
If you fixed an issue with ServiceNow ticketing system, the fix should be in the integration pack.
Please re phrase this :)

Reply:
Done.